runtime/doc/insert.txt

```diff
@@ -1421,9 +1421,9 @@ automatically switch to HTML/CSS/JavaScript completion. Note: contrary to
 original HTML files completion of tags (and only tags) isn't context aware.
 
 
-RUBY							*ft-ruby-omni*
+RUBY						 {Nvim} *ft-ruby-omni*
```

Comment:
Better put `{Nvim}` on a separate line, since it prevents the title `RUBY` from receiving proper title highlighting.